app/components/offers-block.js

```diff
@@ -19,9 +20,26 @@ function safeGet(obj, field, defaultValue) {
   );
 }
 
-export default Component.extend({
+export default Component.extend(cloudinaryImage, {
   store: service(),
 
+  didReceiveAttrs() {
+    this._super(...arguments);
+
+    this.get("items").forEach(item => {
+      const packageTypeName = this.typeOf(item);
+      _.set(item, "packageTypeName", packageTypeName);
```

Comment:
just to keep consistency, instead of using lodash here to set item
better to go with `item.set('packageTypeName', packageTypeName)`

Reply:
item is a js object and not ember object so default way of set and get does not work here hence i used lodash